package.json

```diff
@@ -23,7 +23,7 @@
     "compile": "node ./scripts/build.js",
     "copy": "cd packages/wdio-cli && npm run copy && cd ../../",
     "generate": "run-s generate:*",
-    "generate:bidi": "npx ts-node-esm ./scripts/bidi/index.ts",
+    "generate:bidi": "node --loader ts-node/esm ./scripts/bidi/index.ts",
```

Comment:
I thought these would be the same 🤔

Reply:
That's what I thought too, but:
- `npx ts-node-esm ./scripts/bidi/index.ts` doesn't work on Node 20
- `node --loader ts-node/esm ./scripts/bidi/index.ts` works fine (plus has the added benefit of being consistent with the other scripts)